From a018f82655402421a995565dd4a5192259cbc207 Mon Sep 17 00:00:00 2001
From: Jonathan Scruggs <j.scruggs@gmail.com>
Date: Sat, 23 Sep 2017 10:36:40 +0100
Subject: [PATCH] OpenEXR: Install missing header files

Some header files are not installed via Autotools, but are with
CMake which breaks compatibility with certain programs. This patch
enables Autotools to install these header files.

Signed-off by: Jonathan Scruggs <j.scruggs@gmail.com>
---
 OpenEXR/IlmImf/Makefile.am     |  3 ++-
 OpenEXR/IlmImfUtil/Makefile.am | 17 +++++++++++++++++
 2 files changed, 19 insertions(+), 1 deletion(-)

diff --git a/IlmImf/Makefile.am b/IlmImf/Makefile.am
index a7c219c..b7b96ac 100644
--- a/IlmImf/Makefile.am
+++ b/IlmImf/Makefile.am
@@ -162,7 +162,8 @@ libIlmImfinclude_HEADERS = ImfForward.h ImfAttribute.h ImfBoxAttribute.h \
 			   ImfMisc.h          \
 			   ImfPartHelper.h \
 			   ImfDeepImageState.h \
-			   ImfDeepImageStateAttribute.h
+			   ImfDeepImageStateAttribute.h \
+			   ImfFloatVectorAttribute.h
 
 noinst_HEADERS = ImfCompressor.h    \
 		 ImfRleCompressor.h \
diff --git a/IlmImfUtil/Makefile.am b/IlmImfUtil/Makefile.am
index 8005ee1..e1d3674 100644
--- a/IlmImfUtil/Makefile.am
+++ b/IlmImfUtil/Makefile.am
@@ -33,6 +33,23 @@ libIlmImfUtil_la_LIBADD =  -L$(top_builddir)/IlmImf $(ILMBASE_LIBS) -lIlmImf
 
 libIlmImfUtilincludedir = $(includedir)/OpenEXR
 
+libIlmImfUtilinclude_HEADERS = ImfFlatImage.h \
+			       ImfDeepImage.h \
+			       ImfDeepImageChannel.h \
+			       ImfImageLevel.h \
+			       ImfDeepImageLevel.h \
+			       ImfDeepImageIO.h \
+			       ImfImageChannelRenaming.h \
+			       ImfImageIO.h \
+			       ImfFlatImageChannel.h \
+			       ImfImage.h \
+			       ImfFlatImageLevel.h \
+			       ImfImageDataWindow.h \
+			       ImfSampleCountChannel.h \
+			       ImfFlatImageIO.h \
+			       ImfImageChannel.h
+
+
 EXTRA_DIST = CMakeLists.txt
 
 INCLUDES = \
-- 
2.14.1

